src/routes/comment.routes.js

```diff
@@ -0,0 +1,118 @@
+import express from 'express';
+import comment from '../controllers/comment.controller';
+import commentMiddleware from '../middlewares/comment.validation.middleware';
+import verifyToken from '../middlewares/verify.token.middleware';
+import Validate from '../helpers/validate.helper';
+import isValid from '../middlewares/validate.middleware';
+
+
+const router = express.Router();
+
+/**
+ * @swagger
+ * tags:
+ *   name: comment
+ *   description: Trips activities
+ */
+
+/**
+ * @swagger
+ *
+ * /trip-request/comment/{tripId}:
+ *    post:
+ *      summary: user and manager can comment to the trip request
+ *      tags: [comment]
+ *      parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *       - name: tripId
+ *         in: path
+ *         description: trip request ID
+ *         required: true
+ *         type: string
+ *      requestBody:
+ *        required: true
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/schemas/tripRequestcomment'
+ *      responses:
+ *        "201":
+ *          description: trip request comment schema
+ *
+ * components:
+ *    schemas:
+ *      tripRequestcomment:
+ *        type: object
+ *        required:
+ *          - comment
+ *        properties:
+ *          comment:
+ *            type: string
+ */
+router.post('/comment/:tripId', verifyToken.headerToken, Validate.tripRequestCommentValidation(), isValid, commentMiddleware.validateTipIdAndUserId, comment.createTripRequestComment);
```

Comment:
@shemaeric1234 The better route should be "POST trip-requests/:tripRequestID/comments". It's better because we're creating a comment to/of a trip-request and that trip-request is part of trip-request collections. Kindly update.